1.10/README.md

```diff
@@ -62,6 +62,12 @@ The nginx container image supports the following configuration variable, which c
 | :--------------------- | ----------------------------------------- |
 |  `NGINX_LOG_TO_VOLUME` | When `NGINX_LOG_TO_VOLUME` is set, nginx logs into `/var/opt/rh/rh-nginx110/log/nginx/` |
 
+t your own web root like this:
+```
+$ docker run -v <DIR>:/var/www/html/
```

Comment:
It doesn't seem like this is a working example, there is no image name for example. Also the line above "t your own web root like this:" looks like incomplete.